reconcile/github_repo_invites.py

```diff
@@ -57,11 +69,35 @@ def run(dry_run):
         accept = url in urls or any(url.startswith(org) for org in known_orgs)
         if accept:
             logging.info(["accept", url])
-            invitations.add(url)
+            accepted_invitations.add(url)
 
             if not dry_run:
-                g.accept_repo_invitation(invitation_id)
+                github.accept_repo_invitation(invitation_id)
         else:
             logging.debug(["skipping", url])
+    return accepted_invitations
+
+
+def run(dry_run):
+    gqlapi = gql.get_api()
+    result = gqlapi.query(REPOS_QUERY)
+    settings = queries.get_app_interface_settings()
+    secret_reader = SecretReader(settings=settings)
+
+    # TODO kfischer: This is placed for backwards compatibility.
+    # Once all app-interfaces added the new 'githubRepoInvites' section,
+    # then we can remove this.
```

Comment:
feel free to remove this, this integration is only used in a single app-interface instance (gitlab.cee)

Reply:
Thx for the info! Removed the backwards compat block :+1: